src/openrct2/park/ParkFile.h

```diff
@@ -14,7 +14,7 @@ namespace OpenRCT2
     constexpr uint32_t PARK_FILE_CURRENT_VERSION = 35;
 
     // The minimum version that is forwards compatible with the current version.
-    constexpr uint32_t PARK_FILE_MIN_VERSION = 33;
+    constexpr uint32_t PARK_FILE_MIN_VERSION = 34;
```

Comment:
This is not correct. You need to increase `PARK_FILE_CURRENT_VERSION` and leave this one as-is.